docs/content/en/3.providers.md

```diff
@@ -1,20 +1,22 @@
 ---
 title: Providers
-description: 'Nuxt Image supports multiple providers for high performances.'
-category: 'Getting Started'
+description: "Nuxt Image supports multiple providers for high performances."
+category: "Getting Started"
 ---
 
-## Static Images
+## Default provider
 
-For static projects (`target: 'static'`) image module automatically optimizes `src` for both local and external assets via a preconfigured instance of [IPX](/providers/ipx) during development and generation (`nuxt generate`) so you don't have to worry about choosing a hosting.
+Nuxt Image comes with a preconfigured instance of [IPX](/providers/ipx) to provide images transformations based on the [Sharp](https://www.npmjs.com/package/sharp) package. This default provider can be used without any configuration. The module will automatically optimize `<nuxt-img>` or `<nuxt-picture>` sources and accepts all [options](/api/options/), except for any modifiers that are specific to other providers.
 
-## Image Providers
+With a `server` target, IPX will generate images at runtime. With a `static` target, IPX will generate all optimized assets on running `nuxt generate` and store them in the `dist` folder.
+
+[Learn more about Nuxt deployment targets](https://nuxtjs.org/docs/2.x/features/deployment-targets)
 
-With a server/serverless target or with dynamic images that aren't known at generation-time we need a runtime image provider.
+## Image Providers
 
-Providers are an integration between the image module and image service providers. Using a specific provider means that your images will be transformed into URLs from the image service.
+Providers are integrations between the Nuxt Image module and third-party image transformation services (or, as in the case of `ipx`, a self-hosted/generate-time image transformation service). Each provider is responsible for generating correct URLs for that image transformation service.
```

Comment:
"Transformation service" is not covering imo (also why introducing a new term like this?). Providers, while allowing optional optimization and manipulation/transformation/modification, are for example also CDN provider. A provider is basically URL generator (even for ipx/static)